activity/covid19/items/covid19_vaccine_yesno

```diff
@@ -0,0 +1,38 @@
+{
+    "@context": "https://raw.githubusercontent.com/ReproNim/reproschema/1.0.0-rc2/contexts/generic",
+    "@type": "reproschema:Field",
+    "@id": "covid19_diagnosis",
```

Comment:
same here. please check the `@id` in each file

Reply:
fixed it